### DISCUSSION TITLE
cobbler web

### DISCUSSION THREAD

**Discussion by 1392273211** [cobbler/cobbler-web#256] [Q&A] [answered]
Does the cobbler web3.x series have a corresponding web interface? Can it support the latest cobbler3.3.6?

**Comment by SchoolGuy** [comment]
I am working hard in the recent weeks to progress it. So far there is no fully functional WebUI. If all goes smooth by the end of the month this might change but it will be an initial release that will look very rough and needs a lot of feedback to polish it to a UI that can be productively used.

**Reply by 1392273211** [reply]
Big shot, a version of the cobbler web UI interface will only be available by the end of July 2024. So, will it have more or fewer features than the 2.8.5 version

**Reply by SchoolGuy** [reply]
The idea is that it has the same amount of features. I never used the old UI so far, so I will need to confirm this before making the release.

**Reply by 1392273211** [reply]
I have used the UI of 2.8.5 and it feels good. It would be best if the latest 3.3.6 can support the latest cobbler-web. Does the community have plans to provide a UI interface?

**Reply by SchoolGuy** [reply, marked as answer]
Well considering that I am developing one right now, this is of course the case.

**Reply by 1392273211** [reply]
Thank you for your reply

**Reply by 1392273211** [reply]
If cobbler3.2.2 is already installed, does it mean that there is no web interface? Is the AI ​​interface available?

**Reply by SchoolGuy** [reply]
@1392273211 3.2.2 has the old Django based Web interface.

**Reply by 1392273211** [reply]
Does the cobbler api interface exist or has the function changed between cobbler3.3.4 and 3.3.6

**Reply by SchoolGuy** [reply]
What do you mean by API interface? The XMLRPC API still exists, but the Web UI is a separate project and is independently released. As such no Web UI is generally available at the moment.
